src/main/java/com/minecolonies/achievements/MineColoniesAchievementPage.java

```diff
@@ -10,17 +10,15 @@
  *
  * @since 0.2
  */
-public class MineColoniesAchievementPage extends AchievementPage
-{
+public class MineColoniesAchievementPage extends AchievementPage {
 
     /**
      * Create a new achievement page.
      *
      * @param name         The name this page should have
      * @param achievements A list of achievements to display
      */
-    public MineColoniesAchievementPage(final String name, final Achievement... achievements)
-    {
+    public MineColoniesAchievementPage(final String name, final Achievement... achievements) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)